easybuild/easyblocks/c/cp2k.py

```diff
@@ -792,25 +794,29 @@ def install_step(self):
 
         # copy executables
         targetdir = os.path.join(self.installdir, 'bin')
-        exedir = os.path.join(self.cfg['start_dir'], 'exe/%s' % self.typearch)
+        exedir = os.path.join(self.cfg['start_dir'], 'exe', self.typearch)
         try:
-            if not os.path.exists(targetdir):
-                os.makedirs(targetdir)
-            os.chdir(exedir)
-            for exefile in os.listdir(exedir):
-                if os.path.isfile(exefile):
-                    shutil.copy2(exefile, targetdir)
+            copy_dir(exedir, targetdir)
         except OSError, err:
             raise EasyBuildError("Copying executables from %s to bin dir %s failed: %s", exedir, targetdir, err)
 
+        # copy libraries
+        if self.cfg['library']:
+            targetdir = os.path.join(self.installdir, 'lib')
+            libdir = os.path.join(self.cfg['start_dir'], 'lib', self.typearch, self.cfg['type'])
+            try:
+                copy_dir(libdir, targetdir)
+            except OSError as err:
+                raise EasyBuildError("Copying libraries from %s to lib dir %s failed: %s", libdir, targetdir, err)
```

Comment:
@ocaisa No need for the `try`/`except` anymore here, that's already handled by `copy_dir`; same below